src/main/java/module-info.java

```diff
@@ -0,0 +1,3 @@
+module software.panda.crypto.nacl {
```

Comment:
Note, after merging I corrected this typo `panda` -> `pando`.

@marchof Do you need me to push a new release to Maven Central for you to make use of this?